setup_centos7.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
I didn't look into the script but I think scripts like this should be under build_tools/